typescript/packages/subsurface-viewer/src/layers/grid3d/grid3dLayer.ts

```diff
@@ -85,6 +85,16 @@ async function load_data(
     return Promise.all([points, polys, properties]);
 }
 
+/**
+ * Enumerates possible coloring modes.
+ */
+export enum TGrid3DColoringMode {
```

Comment:
Does this escalate to top level index ?
You can run `npm run doc` to see the warnings...

Reply:
enum TGrid3DColoringMode is now proporgated to the topmost index and the corresponding doc generated.